### DISCUSSION TITLE
Allow inline comment threads in Discussions and Issues

### DISCUSSION THREAD

**Discussion by carlgunderson** [community/community#130437] [Discussions]
### Select Topic Area

Product Feedback

### Body

It would be nice if GitHub Discussions (and Issues) allowed for **inline comments** the same way it does on PRs, where one can comment on a selection of code.

It would **help contextualize conversations** within the Dicussion (or Issue), **reduce endless scrolling** on popular topics, and
> quoting

just to reply to someone.

The current format results in tons of repetitive content that wouldn't be necessary if you allowed threads to reside alongside the source content they refer to.

This could use existing UI components and flows from GitHub PRs, or it could be built fresh and reference implementations from Notion or other productivity apps that support this feature.

I realize there are complications in regards to edits by the maintainers, and probably some other edge cases, but those challenges exist in the other applications too.

Cheers!

**Comment by LiteBrite82** [comment]
Hi @carlgunderson 

We appreciate your feedback and your valuable input. We want to be transparent about our current priorities. You can gain insights into the projects we’re currently focusing on for the upcoming year by checking our [public roadmap](https://github.com/orgs/github/projects/4247). While smaller fixes may not always be explicitly listed, they often get addressed when related work is in progress.

Regarding your specific request, it’s important to note that it’s not actively being worked on at the moment. However, this doesn’t mean we won’t consider it in the future; it’s simply not on our development roadmap right now.

We’d like to emphasize that as a valued GitHub user, your feedback in this Discussion space, along with appropriate tags and examples, plays a crucial role in shaping the development of GitHub. While we can’t guarantee immediate action on every piece of feedback, please rest assured that we are attentive to your suggestions and continuously strive to enhance the GitHub experience.
While we do review all incoming feedback unfortunately due to the volume of feedback, we may not be able to respond to each one individually at this time. However, we strongly encourage you to continue providing your valuable input as it greatly helps us prioritize and improve GitHub.

Your contributions are instrumental in making GitHub better for all users. Thank you for taking the time to share your thoughts with us, even if the particular request is not currently in active development. Your input is truly valuable and contributes to the continuous improvement of GitHub.
